MekHQ/src/mekhq/campaign/stratcon/StratconContractDefinition.java

```diff
@@ -0,0 +1,420 @@
+    // Planetary Assault: Destroy or Capture and Hold X facilities
+    // Relief Duty: Hold X facilities (on hostile world!)
+    // Guerilla Warfare: Victory in X scenarios, soft time limit (reinforcements) on all scenarios
+    // Pirate Hunting: Victory in X scenarios 
+    // Diversionary Warfare: Victory in X scenarios
+    // Objective Raid: Destroy X facilities
+    // Extraction Raid: Extract from X facilities, victory in X capture scenarios
+    // Recon Raid: Recon on X facilities
+    // 
+    // future expansion:
+    // Assassination: Victory in X Assassination scenarios
+    // Terrorism: Victory in X "Riot Suppression" scenarios
+    // Bounty Hunt: Victory in X Assassination, X Capture scenarios
+    // Mole Hunting: ?
+    // Espionage: ?
+    // Sabotage: ?
+    // Observation Raid: ?
+    
+    /**
+     * Serialize this instance of a scenario template to a File
+     * Please pass in a non-null file.
+     * @param outputFile The destination file.
+     */
+    public void Serialize(File outputFile) {
+        try {
+            JAXBContext context = JAXBContext.newInstance(StratconContractDefinition.class);
+            JAXBElement<StratconContractDefinition> templateElement = new JAXBElement<>(new QName(ROOT_XML_ELEMENT_NAME), StratconContractDefinition.class, this);
+            Marshaller m = context.createMarshaller();
+            m.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
+            m.marshal(templateElement, outputFile);
+        } catch(Exception e) {
+            MekHQ.getLogger().error("Erorr deserializing " + outputFile.getPath(), e);
+        }
+    }
+    
+    /**
+     * Attempt to deserialize an instance of a ScenarioTemplate from the passed-in file 
+     * @param inputFile The source file
+     * @return Possibly an instance of a ScenarioTemplate
+     */
+    public static StratconContractDefinition Deserialize(File inputFile) {
+        StratconContractDefinition resultingDefinition = null;
+
+        try {
+            JAXBContext context = JAXBContext.newInstance(StratconContractDefinition.class);
+            Unmarshaller um = context.createUnmarshaller();
+            try (FileInputStream fileStream = new FileInputStream(inputFile)) {
+                Source inputSource = MekHqXmlUtil.createSafeXmlSource(fileStream);
+                JAXBElement<StratconContractDefinition> definitionElement = um.unmarshal(inputSource, StratconContractDefinition.class);
+                resultingDefinition = definitionElement.getValue();
+            }
+        } catch(Exception e) {
+            MekHQ.getLogger().error("Error Deserializing Contract Definition " + inputFile.getPath(), e);
+        }
+
+        return resultingDefinition;
+    }
+}
+
+/**
+ * A manifest containing IDs and file names of scenario template definitions
+ * @author NickAragua
+ *
+ */
+@XmlRootElement(name="contractDefinitionManifest")
+class ContractDefinitionManifest {
```

Comment:
This should either be nested and static or hoisted out into its own file.
```suggestion
private class ContractDefinitionManifest {
```